sos/report/plugins/maas.py

```diff
@@ -11,44 +11,47 @@
 from sos.report.plugins import Plugin, UbuntuPlugin, PluginOpt
 
 
-class Maas(Plugin, UbuntuPlugin):
+class MAAS(Plugin, UbuntuPlugin):
 
-    short_desc = 'Ubuntu Metal-As-A-Service'
+    short_desc = 'MAAS | Metal as a Service'
 
     plugin_name = 'maas'
+    plugin_timeout = 1800
     profiles = ('sysmgmt',)
-    packages = ('maas', 'maas-common')
+
+    packages = (
+        'maas',
+        'maas-region-api',
+        'maas-region-controller',
+        'maas-rack-controller',
+        'maas-agent',
```

Comment:
should we also include maas-netmon or does it make sense to leave it out since it will eventually be part of the agent?

Reply:
The whole ethos here, it is a trigger. So, as long as one of these packages are there, then the plugin would be triggered.

Reply:
Right.
I decided to remove `maas-common` and have an explicit agent/rack/region so it is easier to understand when this plugin is triggered automatically.